sources/models/shared-defaults/aws-creds.toml

```diff
@@ -0,0 +1,20 @@
+[settings.aws]
+profile = "default"
+
+[services.aws]
+configuration-files = [
+  "aws-config",
+  "aws-credentials",
+]
+restart-commands = []
+
+[metadata.settings.aws]
+affected-services = ["aws"]
```

Comment:
I thought the purpose of this was to restart a systemd service, e.g. in this case it would be `aws.service`, when the settings (and thus these files) change. In this case there is no service, right? Is this supposed to be an empty array?

Is this necessary, and if so, should is this the right name (and not e.g. `aws-profile`)?

Reply:
This needs to be an empty array so things are updated but nothing is executed to restart a service.